samples/k8s-existing-cluster-only-sample/mainTemplate.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterResourceId": {
+            "value": "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/sampleresourcegroup/providers/Microsoft.ContainerService/managedClusters/sampleclustername"
```

Comment:
I don't think clusterResourceId would be something ISV need to provide. Deployment test should replace that with their test cluster ID.

Reply:
That's a good point, how should we communicate this to the customer?